go-update/pom.xml

```diff
@@ -117,13 +117,13 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</repository>
-		<repository>
+		<!-- <repository>
```

Comment:
Why is this repository commented out?

Reply:
I think that repository was causing maven to get an "Unauthorized" error when it tried to contact it.